translations/base-tr.yaml

```diff
@@ -120,9 +120,8 @@ dialogs:
         text: "Oyun yükleme başarısız:"
     confirmSavegameDelete:
         title: Silme işlemini onayla
-        text: Are you sure you want to delete the following game?<br><br>
-            '<savegameName>' at level <savegameLevel><br><br> This can not be
-            undone!
+        text: <br><br> kayıdı silmek istiyor musunuz?
+            '<savegameName>'  <savegameLevel><br><br>. seviyede. Bu geri alınamaz!
```

Comment:
- First word in lower case. Should be in upper case.
- Meaning loss in translation: "Bu geri alınamaz!". Could be "Bu **işlem**(process) geri alınamaz!"